aws/logs_monitoring/README.md

```diff
@@ -365,6 +365,10 @@ To test different patterns against your logs, turn on [debug logs](#troubleshoot
 
 `AdditionalTargetLambdaARNs`
 : Comma separated list of Lambda ARNs that will get called asynchronously with the same `event` the Datadog Forwarder receives.
+`InstallAsLayer`
+: Whether to use the layer based installation flow. Set to false to use our legacy installation flow, which installs a second function that copies the forwarder code from Github to an S3 bucket. Disable this option if installing into AWS China regions, where we don't publish layers. Defaults to true.
```

Comment:
Nits:
layer based -> layer-based
Github -> GitHub

Also, this says "Disable this option if installing into AWS China regions, where we don't publish layers." However, looking at the template, it looks like you don't actually need to disable this option when publishing to China -- it happens automatically.